site/topics/fileIO/fileIO.rst

```diff
@@ -0,0 +1,347 @@
+**********************
```

Comment:
Is this file a copy of the original? Should I be reviewing more closely?

Reply:
Nah, just a split at this stage. Will be reworked more later.